src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.Tasks.Task;
+import duke.Tasks.TaskList;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final Scanner scanner;
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public String readCommand() {
+        return this.scanner.nextLine();
+    }
+
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+    }
+
+    public void showLine() {
```

Comment:
Good naming of methods! I can see that you standardize all the method names in most classes! 😄